src/cow-react/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -92,25 +93,40 @@ export function LimitOrdersWidget() {
     label: isWrapOrUnwrap ? undefined : isSellOrder ? 'Your receive at least' : 'You receive exactly',
     currency: outputCurrency,
     rawAmount: isWrapOrUnwrap ? inputCurrencyAmount : outputCurrencyAmount,
-    viewAmount: isWrapOrUnwrap ? inputCurrencyAmount?.toExact() || '' : outputCurrencyAmount?.toExact() || '',
+    viewAmount: isWrapOrUnwrap
+      ? inputCurrencyAmount?.toSignificant(6) || ''
```

Comment:
nit: but you can defien these in a const since you used it above for `viewAmount`

Reply:
Thanks! Will move it to const and function later